workloads/services/k8s/kubeflow-install/kustomize/notebook-controller/overlays/application/culling.yaml

```diff
@@ -1,20 +0,0 @@
-# See GitHub for more details: https://github.com/kubeflow/kubeflow/pull/3856
-# Automatically shutdown Jupyter Notebook containers if idle
```

Comment:
Sad to see this go. Any idea if the ability to automatically cull idle gpu jobs still exists?